app/components/funding/view.rb

```diff
@@ -29,30 +29,49 @@ def funding_detail_rows
 
     attr_accessor :data_model, :has_errors
 
+    delegate :can_apply_for_scholarship?, :scholarship_amount,
+             :can_apply_for_bursary?, :bursary_amount,
+             to: :funding_manager
+
     def training_initiative_row
-      mappable_field(training_initiative, t(".training_initiative"), edit_trainee_funding_training_initiative_path(trainee))
+      mappable_field(
+        training_initiative,
+        t(".training_initiative"),
+        edit_trainee_funding_training_initiative_path(trainee),
+      )
     end
 
     def funding_method_row
+      if data_model.applying_for_scholarship
+        scholarship_funding_row
+      else
+        bursary_funding_row
```

Comment:
Do we also need a guard for `applying_for_bursary`?

Reply:
i thought the same thing, but then realised that logic is contained within the `bursary_funding_row` itself (see `def funding_method`)

could be refactored to be easier to follow for sure.

Reply:
Cool 😎 next time refactoring will be nice